owslib/iso.py

```diff
@@ -298,22 +302,39 @@ def __init__(self, md=None):
             self.role = _testCodeListValue(md.find(util.nspath_eval('gmd:role/gmd:CI_RoleCode', namespaces)))
 
 
+class keyword(object):
```

Comment:
Use CamelCase for class names.

Reply:
Camel Case added in 1d5df8a, I will look to add a test now too

Reply:
Tests added in 72c4148